cli/cmd/sync.go

```diff
@@ -302,7 +302,8 @@ func sync(cmd *cobra.Command, args []string) error {
 
 func filterPluginEnv(environ []string, pluginName, kind string) []string {
 	env := make([]string, 0)
-	prefix := "__" + kind + "_" + pluginName + "__"
+	cleanName := strings.ReplaceAll(pluginName, "-", "_")
+	prefix := strings.ToUpper("__" + kind + "_" + cleanName + "__")
```

Comment:
Does this need a backend change too?

Reply:
Yes, but we have the CLI version pinned in the backend so I will make an update that upgrades and makes the corresponding change there shortly